pkg/sentry/csr/csr.go

```diff
@@ -130,6 +137,33 @@ func GenerateCSRCertificate(csr *x509.CertificateRequest, subject string, signin
 	cert.Extensions = csr.Extensions
 	cert.BasicConstraintsValid = true
 	cert.SignatureAlgorithm = csr.SignatureAlgorithm
+
+	if identityBundle != nil {
+		spiffeID, err := identity.CreateSPIFFEID(identityBundle.TrustDomain, identityBundle.Namespace, identityBundle.ID)
+		if err != nil {
+			return nil, errors.Wrap(err, "error generating spiffe id")
+		}
+
+		rv := []asn1.RawValue{
+			{
+				Bytes: []byte(spiffeID),
+				Class: asn1.ClassContextSpecific,
+				Tag:   6,
```

Comment:
I would use constant `asn1.TagOID` defined in encoding/asn1.

Reply:
Changed.